tests/bindings/python/test_units.py

```diff
@@ -0,0 +1,71 @@
+#
+# Tests the Units class bindings
+#
+import sys
+import unittest
+
+class UnitsTestCase(unittest.TestCase):
+
+    def test_model(self):
+        from libcellml.units import Units
+        
+        # Test create/copy/destroy
+        x = Units()
+        del(x)
+        y = Units()
+        z = Units(y)
+        del(y,z)
+        
+        # Test inheritance
+        x = Units()
+        from libcellml.importedentity import ImportedEntity
+        self.assertIsInstance(x, ImportedEntity)
+        from libcellml.namedentity import NamedEntity
+        self.assertIsInstance(x, NamedEntity)
+        from libcellml.entity import Entity
+        self.assertIsInstance(x, Entity)
+        
+        # Test access to inherited methods
```

Comment:
Again, `test_method_inheritance` or something to that effect, and so on so forth.